src/c++/perf_analyzer/genai-pa/genai_pa/parser.py

```diff
@@ -202,14 +225,17 @@ def parse_args(argv=None):
     parser.set_defaults(func=handler)
 
     # Conceptually group args for easier visualization
-    add_model_args(parser)
-    add_profile_args(parser)
-    add_endpoint_args(parser)
-    add_dataset_args(parser)
+    _add_model_args(parser)
+    _add_profile_args(parser)
+    _add_endpoint_args(parser)
+    _add_dataset_args(parser)
 
-    args = parser.parse_args(argv)
+    args, extra_args = parser.parse_known_args(argv)
```

Comment:
@debermudez @rmccorm4 I think we should print a warning if there are known args detected. This serves a few purposes:
1. Notifies the user that this mode is being used. If this mode is only for power users or those who want to go beyond the supported arguments, they should get a warning letting them know that this is what's happening and is not directly supported.
2. Avoids errors getting becoming unclear, e.g. if you call "genai-pa -- model-name <model>", you might not realize you are entering this mode, especially if it is not documented (or not documented with the basic instructions).
3. Makes the code self-commenting. This is less of an issue if using parse_known_args, which is documented, but having this logging statement lets anyone looking at this code (developers or users) know that the intent is to allow power users to feed PA arguments directly, even if those arguments are not yet supported by GenAi-PA.

Reply:
If you expect the python script argparsing to handle the common cases, and this passthrough arg support is just for edge cases, then I agree it would be useful to call out with a log statement.

Reply:
Adding a logging statement